WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayTryPaymentPromptViewController.swift

```diff
@@ -0,0 +1,182 @@
+                .padding()
+                .fixedSize(horizontal: false, vertical: true)
+
+            Text(Localization.setUpTryPaymentPromptDescription)
+                .font(.body)
+                .multilineTextAlignment(.center)
+                .padding()
+                .fixedSize(horizontal: false, vertical: true)
+
+            Spacer()
+
+            Button(Localization.tryAPaymentButton, action: {
+                viewModel.tryAPaymentTapped()
+            })
+            .buttonStyle(PrimaryLoadingButtonStyle(isLoading: viewModel.loading))
+
+            Button(Localization.skipButton, action: {
+                viewModel.skipTapped()
+            })
+            .buttonStyle(SecondaryButtonStyle())
+
+            LazyNavigationLink(destination: paymentFlow(), isActive: $viewModel.summaryActive) {
+                EmptyView()
+            }
+        }
+        .padding()
+    }
+
+    /// Returns a `SimplePaymentsSummary` instance when the view model is available.
+    ///
+    private func paymentFlow() -> some View {
+        WooNavigationSheet(viewModel: WooNavigationSheetViewModel(navigationTitle: Localization.setUpTryPaymentPromptTitle,
+                                                                  done: viewModel.dismiss ?? {})) {
+            if let summaryViewModel = viewModel.summaryViewModel {
+                SimplePaymentsSummary(
+                    dismiss: {
+                        showOrder(orderID: summaryViewModel.orderID, siteID: summaryViewModel.siteID)
+                    },
+                    rootViewController: rootViewController?.navigationController,
+                    viewModel: summaryViewModel.simplePaymentSummaryViewModel)
+            }
+            EmptyView()
+        }
+    }
+
+    private func showOrder(orderID: Int64, siteID: Int64) {
+        let orderViewController = OrderLoaderViewController(orderID: orderID, siteID: siteID)
+        orderViewController.addCloseNavigationBarButton(title: Localization.doneButton)
+        rootViewController?.navigationController?.navigationBar.isHidden = false
+        rootViewController?.navigationController?.setViewControllers([orderViewController], animated: true)
+    }
+}
+
+private enum Constants {
+    static let maxImageHeight: CGFloat = 180
+    static let maxCompactImageHeight: CGFloat = 80
+    static let imageFontSize: CGFloat = 120
+    static let compactImageFontSize: CGFloat = 40
+}
+
+// MARK: - Localization
+//
+private extension SetUpTapToPayPaymentPromptView {
+    enum Localization {
+        static let setUpTryPaymentPromptTitle = NSLocalizedString(
+            "Try a payment",
+            comment: "Settings > Set up Tap to Pay on iPhone > Try a Payment > Inform user that " +
+            "Tap to Pay on iPhone is ready"
+        )
+
+        static let setUpTryPaymentPromptDescription = NSLocalizedString(
+            "Try taking a payment of $0.50 to see how Tap to Pay on iPhone works. Use your " +
```

Comment:
Non-blocker: 

```suggestion
            "Try taking a payment of USD $0.50 to see how Tap to Pay on iPhone works. Use your " +
```
While we support only USD at the moment, using `$` can cause confusion to folks that may use the same symbol for their currency, this happens pretty often for example for people that has their account in MXN and they're prompted to pay for a plan in USD.

Reply:
This will only show for US stores, even though I've not put that constraint on yet. In that context I don't think it should be too confusing?